dev/build.image/profiles/jakartaee9/features.xml

```diff
@@ -1,5 +1,7 @@
 <feature>jakartaee-9.0</feature>
 <feature>jakartaeeClient-9.0</feature>
+<feature>jakartaee-9.1</feature>
+<feature>jakartaeeClient-9.1</feature>
 <feature>appSecurityClient-1.0</feature>
 <feature>localConnector-1.0</feature>
 <feature>transportSecurity-1.0</feature>
```

Comment:
Should we use this PR to add the other features that are part of javaee-8.0's package?
```
<feature>monitor-1.0</feature>
<feature>ldapRegistry-3.0</feature>
<feature>sessionCache-1.0</feature>
<feature>sessionDatabase-1.0</feature>
```

Reply:
Of course adding the last 3 of those will have the issues discussed with the webProfile package.